lib/Doctrine/Common/Annotations/DocParser.php

```diff
@@ -421,12 +423,8 @@ private function matchAny(array $tokens)
      *
      * @param string     $expected Expected string.
      * @param array|null $token    Optional token.
```

Comment:
the descriptions don't really provide any value here, and the types are not specified through typehints